src/sensorthings.js

```diff
@@ -22,9 +27,9 @@ module.exports = (config) => {
   let router = express.Router();
   router.use('/', baseRouter);
 
-  resourceEndpoints.forEach(endpoint => {
-    router.use(route.generate(endpoint),
-               require('./routes/' + Case.snake(endpoint)));
+  entities.forEach(endpoint => {
+    router.use(route.generate(endpoint), associations,
```

Comment:
route.generate() (without params) generates a general URL that matches any correct URL. If we do something like:

`router.use(route.generate(), associations);`

just once before the loop, I think that would be enough.